Fixing format of the known issue and making imporvements (#7898)
open-cluster-management/rhacm-docs: release_notes/known_issues_install.adoc

When you upgrade to {acm-short} 2.13 with an {ocp-short} version 4.18 cluster, you receive an error installing the CRD and remain in a `stuck` status. 

To resolve this issue, upgrade directly to {acm-short} 2.13.2, even if you upgraded to 2.13.1 and receive this error. Complete the following steps:

. To get more information about the error, run the following command:

+
[source,bash]
----
oc get multiclusterhubs -n open-cluster-management -o yaml
----
+

> The following error message about a failed installation of `multiclusterobservabilities.observability.open-cluster-management.io` is displayed from your hub cluster:

Author: ```suggestion
The hub cluster reports the following error message that shows installing `multiclusterobservabilities.observability.open-cluster-management.io` failed:
```

> You might receive the following error message about a failed installation of `multiclusterobservabilities.observability.open-cluster-management.io` on your hub cluster:

Author: Will discuss this update, willing to revert if need be. The main issue was that actions were not listed in an ordered list so alignment became incorrect

> You might receive the following error message about a failed installation of `multiclusterobservabilities.observability.open-cluster-management.io` on your hub cluster:

Author: ```suggestion
The following error message about a failed installation of `multiclusterobservabilities.observability.open-cluster-management.io` is displayed from your hub cluster:
```
